src/util.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::Result;
+use osmpbfreader::{OsmId, OsmObj, OsmPbfReader};
+use std::{collections::BTreeMap, path::PathBuf};
+
+/// Filter for relations havng name and a range of `admin_level`.
+pub fn filter_all_relations(obj: &OsmObj) -> bool {
+    obj.is_relation()
+}
+
+/// Filter for relations. In addition to `filter_all_relations`, add boundary types.
+pub fn filter_target_relations(obj: &OsmObj) -> bool {
+    let tags = obj.tags();
```

Comment:
Note: das binding hier macht den code nur leserlicher, sollte aber in release builds keinen schnelleren code generieren, weil der optimizer ziemlich sicher [den sehr einfachen code für tags](https://github.com/TeXitoi/osmpbfreader-rs/blob/master/src/objects.rs#L141-L147) inlinen und so optimieren würde, dass `obj.tags()` einfach die Adresse der tags enthalten würde.

Reply:
Ah, gut zu wissen!